src/plugins/dashboard/public/services/dashboard_content_management/lib/check_for_duplicate_dashboard_title.ts

```diff
@@ -30,6 +35,7 @@ export async function checkForDuplicateDashboardTitle(
     lastSavedTitle,
     onTitleDuplicate,
     isTitleDuplicateConfirmed,
+    searchLimit = 20,
```

Comment:
Why does this need to be an argument? It seems like it can just be hardcoded in the function more simply.